config/stage.json

```diff
@@ -21,6 +21,7 @@
 		"community-translator": true,
 		"desktop-promo": true,
 		"guided-tours": true,
+		"happychat": true,
```

Comment:
We usually first turn on wpcalypso / horizon before testing in staging. Enabled in development as early as possible, and then in wpcalypso as soon as it doesn't break other areas. Once it's ready, we enable in staging and do final testing.